### ISSUE TITLE
Declaration of Frapi_Response_Custom::setData() should be compatible with that of Frapi_Response::setData()

### ISSUE THREAD

**Issue by omissis** [frapi/frapi#54]
Hi I have found this issue on my local environment(osx, apache 2.2, php 5.3.2) right after I've completed the setup:

Strict Standards: Declaration of Frapi_Response_Custom::setData() should be compatible with that of Frapi_Response::setData() in FRAPI_PATH/frapi/src/frapi/library/Frapi/Response/Custom.php on line 34

Fatal error: Uncaught exception 'Zend_Session_Exception' with message 'Session must be started before any output has been sent to the browser; output started in FRAPI_PATH/frapi/src/frapi/library/Frapi/Response/Custom.php/34' in FRAPI_PATH/frapi/src/frapi/library/Zend/Session.php:454 

Stack trace: 
#0 FRAPI_PATH/frapi/src/frapi/library/Zend/Session/Namespace.php(143): Zend_Session::start(true)
#1 FRAPI_PATH/frapi/src/frapi/admin/application/Bootstrap.php(31): Zend_Session_Namespace->__construct('locale')
#2 FRAPI_PATH/frapi/src/frapi/library/Zend/Application/Bootstrap/BootstrapAbstract.php(662): Bootstrap->_initLanguages()
#3 FRAPI_PATH/frapi/src/frapi/library/Zend/Application/Bootstrap/BootstrapAbstract.php(615): Zend_Application_Bootstrap_BootstrapAbstract->_executeResource('languages')
#4 FRAPI_PATH/frapi/src/frapi/library/Zend/Application/Bootstrap/BootstrapAbstract.ph in FRAPI_PATH/frapi/src/frapi/library/Zend/Session.php on line 454

This is due to a little error(type hinting's missing) in overriding setData() method. Here's the patch to fix the problem: 

```
diff --git a/src/frapi/library/Frapi/Response/Custom.php b/src/frapi/library/Frapi/Response/Custom.php
index b34a194..10255dd 100644
--- a/src/frapi/library/Frapi/Response/Custom.php
+++ b/src/frapi/library/Frapi/Response/Custom.php
@@ -27,7 +27,7 @@ class Frapi_Response_Custom extends Frapi_Response
      * @param  string $data The data to set in the response.
      * @return void
      */
-    public function setData($data)
+    public function setData(array $data)
     {
         $this->data = $data;
     }
```

**Comment by davidcoallier** [comment]
Closed by 56930dd377365aae9df6f30055c916a8d6ed8999: Declaration of setData breaks E_STRICT

**Comment by davidcoallier** [comment]
Thank you for the report. The issue has been fixed and pushed to the master branch.

https://github.com/frapi/frapi/commit/56930dd377365aae9df6f30055c916a8d6ed8999
